src/chapters/orm-relationships/many-to-many.rst

```diff
@@ -0,0 +1,690 @@
+
+.. todo: Add display tags video
+
+Displaying Tags in the Detail View - Text
+-----------------------------------------
+
+We want an event's tags to be displayed on its detail view, so let's start in ``EventsController``. The ``Detail`` method needs to pass in tag data for the given event. To do this, we must query ``context.EventTags`` and pass the resulting list of ``EventTag`` objects into the ViewModel's constructor.
+
+.. sourcecode:: csharp
+   :lineno-start: 92
+
+   List<EventTag> eventTags = context.EventTags
+         .Where(et => et.EventId == id)
+         .Include(et => et.Tag)
+         .ToList();
+
+   EventDetailViewModel viewModel = new EventDetailViewModel(theEvent, eventTags);
+
+Our query of ``context.EventTags`` has a few pieces:
+
+#. **Line 93** - Filters the ``EventTags`` set to include only objects related to the given ``Event``.
+#. **Line 94** - Forces eager loading of the ``Tag`` property of those ``EventTag`` objects.
+#. **Line 95** - Converts the ``DbSet`` to a list.
+
+.. admonition:: Note
+
+   You might be wondering why we have to query ``context.EventTags``. Indeed, it would be convenient of we could just reference a ``Tags`` property from the ``Event`` class. But notice that there *is no such property* in ``Event``. The many-to-many relationship is defined by the data in ``EventTag``, so we must use this class in order to access related objects.
+
+Now let's move into ``EventDetailsViewModel``. Here, we can add data related to an event's tags, so that it can be passed into the view.
+
+First, add a new string property named ``TagText``.
+
+.. sourcecode:: csharp
+   :lineno-start: 14
+
+   public string TagText { get; set; }
+
+Then, in the constructor add a parameter to represent the list of all of ``EventTag`` objects associated with a given ``Event``. Use this parameter to set the value of ``TagText``.
+
+.. sourcecode:: csharp
+   :lineno-start: 16
+
+   public EventDetailViewModel(Event theEvent, List<EventTag> eventTags)
+   {
+      EventId = theEvent.Id;
+      Name = theEvent.Name;
+      Description = theEvent.Description;
+      ContactEmail = theEvent.ContactEmail;
+      CategoryName = theEvent.Category.Name;
+
+      TagText = "";
+      for (var i = 0; i < eventTags.Count; i++)
+      {
+            TagText += ("#" + eventTags[i].Tag.Name);
+            if (i < eventTags.Count - 1)
+            {
+               TagText += ", ";
+            }
+      }
+   }
+
+We build up the contents of ``TagText`` by looping over ``eventTags`` and appending tag names, separated by commas. For example, if an event has tags with names ``"java"``, ``"csharp", and ``"object-oriented"``, then the ``TagList`` will be ``"#java, #csharp, #object-oriented"``. 
```

Comment:
the syntax highlighting is a little off here but i think you can get away with removing it completely from ``"java"``, ``"csharp", and ``"object-oriented"``